R/transect-weighting.R

```diff
@@ -32,15 +32,21 @@ transect_weighting <- function(riverkm, spawning, rkm) {
 #'   Spawning = c(0.1,0.8,0.1)
 #' ))
 edw_transect_weighting <- function(rkm, data, riverkm = "RiverKm", spawning = "Spawning") {
-  check_vector(rkm, 1, unique = TRUE)
+  chk_vector(rkm)
+  check_values(rkm, 1)
+  chk_unique(rkm)
   
-  check_data(data, c(riverkm, spawning), nrow = TRUE)
+  check_data(data, nrow = TRUE)
+  check_names(data, names = c(riverkm, spawning))
   
-  check_vector(data[[riverkm]], 1, unique = TRUE, x_name =
-                 paste0("column '", riverkm, "' of data"))
+  chk_unique(data[[riverkm]])
```

Comment:
The x_name should be `paste0("column '", riverkm, "' of data")` for all the `data[[riverkm]]` checks